Sources/Wrap.swift

```diff
@@ -340,11 +344,13 @@ extension NSArray: WrapCustomizable {
 }
 
 /// Extension customizing how NSDictionaries are wrapped
-extension NSDictionary: WrapCustomizable {
-    public func wrap(context: Any?, dateFormatter: DateFormatter?) -> Any? {
-        return try? Wrapper(context: context, dateFormatter: dateFormatter).wrap(dictionary: self as [NSObject : AnyObject])
+#if os(OSX) || os(iOS) || os(watchOS) || os(tvOS)
```

Comment:
I'll admit its simpler, but its also more fragile. Eventually (fingers crossed) Swift will be available on other platforms than Apple & Linux (think Windows, FreeBSD, Android etc.). I try to make my code future proof, hoping that Swift will be available on other platforms sooner rather than later.